raft.go

```diff
@@ -438,6 +456,8 @@ func nonInitNode(args *NodeConfig) (*Node, error) {
 
 	if args.SnapshotConfig == nil {
 		args.SnapshotConfig = DefaultSnapshotConfig
+	} else if args.SnapshotConfig.MaxRetainedSnapshots == 0 {
```

Comment:
@doodles526 

Q. then shouldn't `MaxRetainedSnapshots` be specified as 3 in `DefaultSnapshotConfig`?
I see that field is empty, so 0? etcd `DefaultMaxSnapshots` is 5.